qa/workunits/cephadm/test_iscsi_pids_limit.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```

Comment:
You might want to add this

```suggestion
set -ex
```

Reply:
Please do it the way @guits suggested, so that `-ex` is applied independent of how the script is run.  Since you are using plain `test` (i.e. no explicit `exit` with non-zero status), `-ex` is crucial.

Reply:
@idryomov I added -ex to bash instead of adding a new line with set -ex. Do you prefer to have set -ex instead of bash -ex ?

Reply:
Yes.  As I said, this is to ensure that `-ex` is still applied if the script is invoked as e.g. `bash test_iscsi_pids_limit.sh` instead of as `./test_iscsi_pids_limit.sh`.

Reply:
change applied but we should do the same on all the files in qa/workunits/cephadm/* as they all use bash -ex.